src/network/hub.rs

```diff
@@ -261,6 +261,7 @@ impl Server<HubData> for HubData {
             "get_joint" => ws.on_get_joint(params)?,
             "catchup" => ws.on_catchup(params)?,
             "get_hash_tree" => ws.on_get_hash_tree(params)?,
+            "get_witnesses" => ws.on_get_witnesses()?,
```

Comment:
we should keep consistence with other handler functions signature, although we don't use the param here